services/brig/src/Brig/API.hs

```diff
@@ -1511,10 +1508,9 @@ verifyDeleteUserH (r ::: _) = do
   API.verifyDeleteUser body !>> deleteUserError
   return (setStatus status200 empty)
 
-onboardingH :: JSON ::: UserId ::: JsonRequest AddressBook -> Handler Response
-onboardingH (_ ::: uid ::: r) = do
-  ab <- parseJsonBody r
-  json <$> API.onboarding uid ab !>> connError
+-- | DEPRECATED / does nothing.
+onboardingH :: JSON ::: UserId ::: JsonRequest Value -> Handler Response
```

Comment:
I think we still want to return `{"results":[],"auto-connects":[]}`, so clients don't choke on the response. 
So we could either leave the `MatchingResult` type and do `pure $ MatchingResult [] []` here, or just produce a JSON Value with the right fields.

Maybe this structure should also still be documented in Swagger.